ui/components/DataTable.tsx

```diff
@@ -616,6 +616,9 @@ export const DataTable = styled(UnstyledDataTable)`
     overflow: hidden;
     text-overflow: ellipsis;
   }
+  .filterOptionsChip {
```

Comment:
css conventions is usually to use kebab-case. We don't have this documented anywhere though. Is it mixed up around the place?

```suggestion
  .filter-options-chip {
```